checkout/action.yaml

```diff
@@ -39,4 +39,10 @@ runs:
       path: ${{ inputs.path }}
       ref: ${{ inputs.ref }}
       repository: ${{ inputs.repository }}
-      set-safe-directory: true
+      # set-safe-directory: true <- not sure what the point is since this doesn't persist beyond the "Checkout" step
```

Comment:
I ran into this to, look like it's a bug with docker containers, https://github.com/actions/checkout/issues/1169